TidepoolServiceKitUI/SettingsView.swift

```diff
@@ -177,6 +181,17 @@ public struct SettingsView: View {
         .disabled(isLoggingIn)
     }
 
+    private var continueButton: some View {
+        Button(action: {
+            dismiss?()
+        }) {
+            Text(LocalizedString("Continue", comment: "Delete Tidepool service button title"))
```

Comment:
FYI, you don't need to use a LocalizedString inside a `Text` initializer. You can do this instead to the same effect:

```suggestion
            Text("Continue", comment: "Delete Tidepool service button title")
```

Reply:
We use `LocalizedString` in frameworks of Loop, so that the lookup happens from the *framework's* bundle. You'll see this isn't NSLocalizedString, but a function local to the framework.

Reply:
Ahh, got it!